cloud/providers/aws/create.go

```diff
@@ -71,6 +71,12 @@ func (cm *ClusterManager) SetDefaultCluster(cluster *api.Cluster, config *api.Cl
 		"cloud-provider": cluster.Spec.Config.Cloud.CloudProvider,
 	}
 
+	if cluster.Spec.AuditSink {
+		cluster.Spec.Config.APIServerExtraArgs["audit-dynamic-configuration"] = "true"
+		cluster.Spec.Config.APIServerExtraArgs["feature-gates"] = "DynamicAuditing=true"
```

Comment:
How are we going to handle enabling multiple feature gates?

Reply:
Probably using comma separated values.
```
cluster.Spec.Config.APIServerExtraArgs["feature-gates"] = "DynamicAuditing=true,AnotherFeatureGate=true"
```
We can make a type `FeatureGate (map[string]bool)` . Which should have a method that will convert the FeatureGate values to comma separated values.

ref: https://github.com/pharmer/pharmer/blob/a0c0f090db0c100700cf7b804f9ffea799db5680/cloud/providers/aws/startupscript.go#L38